core/client/src/test/java/alluxio/client/file/FileOutStreamTest.java

```diff
@@ -97,11 +104,13 @@
   @Before
   public void before() throws Exception {
     ClientTestUtils.setSmallBufferSizes();
+    mDelegationEnabled = Configuration.getBoolean(PropertyKey.USER_UFS_DELEGATION_ENABLED);
```

Comment:
I just meant to use it to make it explicit that the test expects it to be enabled. I'm fine either way though so I'll leave it to you